CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Current Master
 
+- Removed network Id 2 from the explorer links and added network Id 3
```

Comment:
Make this change message user-facing.  What does a normal person care about this change?

Maybe "Fixed ropsten transaction links"?